src/main/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPlugin.kt

```diff
@@ -344,7 +365,8 @@ class NexusPublishPlugin : Plugin<Project> {
     }
 }
 
-inline fun <reified T : Any> Project.theExtension(): T =
+private inline fun <reified T : Any> Project.theExtension(): T =
```

Comment:
I think this is visible in RC1 only, it kind-of pollutes global namespace if someone has similarly named utilities.

Reply:
:+1: